_rules/text-contrast-afw4f7.md

```diff
@@ -316,22 +358,6 @@ This text not part of a [text node][].
 
 #### Inapplicable Example 6
 
-This text is part of a widget because it is a child of a `button` element.
```

Comment:
It would be good to have a disable button example in here. Maybe one with the native `disabed` attribute, and an ARIA one with `aria-disabled=true`?

Reply:
@WilcoFiers Added Inapplicable 10 and 11. Is this what you were expecting?